testpilot/frontend/static-src/app/views/landing-page.js

```diff
@@ -5,6 +5,8 @@ import ExperimentListView from './experiment-list-view';
 
 // TODO back out this commit (78bd0a93a1q) before launch.
 import queryString from 'query-string';
+const installedMsgTmp = `<h2 class="emphasis">Test Pilot installed!</h2>
+<a href="/" class="button large default quick-pop">Let's Go</a>`;
```

Comment:
I'm not 100% certain on our l20n flow, but will these work with our localization flow, or is this just temporary text?